site-src/implementations/gateways.md

```diff
@@ -8,13 +8,15 @@ This project has several implementations that are planned or in progress:
 * [Google Kubernetes Engine][4]
 * [Istio][5]
 * [Kgateway][6]
+* [Kubvernor Rust API Gateway][7]
 
 [1]:#agentgateway
 [2]:#alibaba-cloud-container-service-for-kubernetes
 [3]:#envoy-ai-gateway
 [4]:#google-kubernetes-engine
 [5]:#istio
 [6]:#kgateway
+[7]:#kubvernor-rust-api-gateway
```

Comment:
Please remove the project description and use the project name.

Reply:
Done